includes/Core/Consent_Mode/REST_Consent_Mode_Controller.php

```diff
@@ -150,10 +150,11 @@ protected function get_rest_routes() {
 					array(
 						'methods'             => WP_REST_Server::READABLE,
 						'callback'            => function () {
-							$is_active = function_exists( 'wp_set_consent' );
-							$installed = $is_active;
-							$slug      = 'wp-consent-api';
-							$plugin    = "$slug/$slug.php";
+							$is_active  = function_exists( 'wp_set_consent' );
```

Comment:
@10upsimon, was it added by accident?

Reply:
Thanks @eugene-manuilov definitely by accident from research into another issue, I must have not stashed and accidentally added. Good catch, thank you!

Reply:
@eugene-manuilov back with you